tests/SixLabors.Fonts.Tests/SystemFontCollectionTests.cs

```diff
@@ -156,6 +156,11 @@ public void SystemFonts_FontFamilyNotFound_ThrowsWithSearchDirectories()
                 Assert.Contains(@"/Library/Fonts/", exception.Message, StringComparison.OrdinalIgnoreCase);
                 Assert.Contains(exception.SearchDirectories, e => e.IndexOf("Library/Fonts", StringComparison.OrdinalIgnoreCase) != -1);
             }
+            else if (RuntimeInformation.IsOSPlatform(OSPlatform.Create("Android")))
```

Comment:
We don't have the infrastructure in place to test this but definitely good to add for the future.